src/common/connect.js

```diff
@@ -103,8 +103,7 @@ export default vdom => {
 			}
 
 			this.render = () => {
-				let { form: Form } = context;
```

Comment:
can you explain why you are deleting this line? Shouldn't it get the most up to date copy of Form from context?